src/html/index.html

```diff
@@ -3,35 +3,33 @@
 <div class="o-wrapper">
   <div class="o-layout">
     <div class="o-layout__item">
-      <h1 data-module="example">{{ message }}</h1>
-      <h2 data-module="example">{{ message }}</h2>
-      <h3 data-module="example">{{ message }}</h3>
-      <p data-module="example">{{ message }}</p>
 
-      <div style="background-color:#4a4a4a; padding:10px; height:auto">
-        <button class="c-btn  c-btn--primary">Register</button>
-        <button class="c-icon  c-icon--huge  c-icon--darkbg">{{sprite("play")}}</button>
+      {% include "./shared/share.html" %}
 
-        <div class="c-watch">
-          <button class="c-watch--btn  c-icon  c-icon--small  c-icon--darkbg">{{sprite("play")}}
-            <span class="c-watch--text">watch the video</span>
-          </button>
-        </div>
-      </div>
 
-      <div style="background-color:#fbfbfb; padding:10px; height:auto">
-        <button class="c-btn  c-btn--secondary">Register</button>
-        <button class="c-icon  c-icon--huge  c-icon--lightbg">{{sprite("play")}}</button>
+    </div>
+  </div>
 
-        {% include "./shared/share.html" %}
 
 
-      </div>
 
-    </div>
-  </div>
+</div>
 
+<nav class="c-footer-bottom">
+  <div class="o-wrapper">
+    <ul class="o-list-bare o-list-inline">
+      <li class="o-list-bare__item o-list-inline__item c-footer-bottom--item">
+        <a class="c-text--reset" href="#" target="_blank">About</a>
```

Comment:
Rather than using `.c-text--reset` I'd recommend just targeting `a` explicitly as a child of `.c-footer-bottom` to remove text-decoration etc.

`.c-text--reset` isn't a bad idea necessarily, but would be better suited to a utility class if used.